pkg/unvanquished_src.dpkdir/ui/options_ui.rml

```diff
@@ -4,8 +4,21 @@
 		<link type="text/template" href="/ui/shared/window.rml" />
 	</head>
 	<body id="options_ui" template="window" style="width: 32em; margin: 1em;">
-		<h1><translate>User interface</translate></h1>
-
+		<row>
+			<h1><translate>User interface</translate></h1>
+			<h3><translate> Language</translate> </h3>
```

Comment:
Remove leading spaces inside of `<translate>` tags

Reply:
Who are you talking to?

Reply:
Actually that leading space will be included in the translation string, this leading space is not ignored.

Reply:
Sure. Either fix it or don't, do whatever you want. But attempts to command me to work are completely inappropriate.

Reply:
I agree.

Reply:
The PR is pointed at a branch on your personal repository so no one else can edit it.

Reply:
Does "Allow edits by maintainers" not work?

Reply:
I believe that refers to the messages in the comment thread. Not the code.

Reply:
Done by me